pom.xml

```diff
@@ -4,7 +4,7 @@
 
     <groupId>com.salesforce.mirus</groupId>
     <artifactId>mirus</artifactId>
-    <version>0.0.13-SNAPSHOT</version>
+    <version>0.0.14-SNAPSHOT</version>
```

Comment:
You should not modify this value.  Since we're not being hobbled by Strata, Mirus is able to use the normal release plugin and release versioning is all handled for us.  The current release is 0.0.12, so we develop on 0.13-SNAPSHOT.